config/crd/bases/awx.ansible.com_awxmeshingresses.yaml

```diff
@@ -69,6 +69,11 @@ spec:
               ingress_annotations:
                 description: Annotations to add to the Ingress Controller
                 type: string
```

Comment:
@sharvarikhedkar we need a similar entry in the CSV file

Reply:
I'm not sure I follow, do you mean https://github.com/ansible/awx-operator/pull/2045/files#diff-2a4968792da0598da409c1e5bbc604542cdf7b2b52bbbf4179dd018fcf9baaf4R53-R57?

Reply:
I missed that on the first pass.  Looks like you have it already, thanks.  Resolved.